### PR TITLE
fix(svLogo): Fix file name

### PR DESCRIPTION
**Pull request by JCB-K**
- Correct file name so project will compile
- Correct width/height ratio
- More consistent class names

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Pane from './components/pane/pane';
 import ProgressBar from './components/progress-bar/progress-bar';
 import RadioGroup from './components/radio-group';
 import SegmentedControl from './components/segmented-control';
-import SharevilleLogo from './components/shareville-logo';
+import SharevilleLogo from './components/shareville-logo/shareville-logo';
```
Comment (iamstarkov) [inline src/index.js]: could you have index.js file, where you re-export stuff, so components' declarations will be consistent?

File: test/components/shareville-logo/logo.test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('<SharevilleLogo />', () => {
   });
 
   it('should render a SharevilleLogo', () => {
-    expect(wrapper.find('.sharevilleLogoWTClass-1')).to.have.length(1);
+    expect(wrapper.find('.svLogoGradientWtClass-1')).to.have.length(1);
```
Comment (iamstarkov) [inline test/components/shareville-logo/logo.test.js]: nay

Comment (iamstarkov) [reply test/components/shareville-logo/logo.test.js]: please use mockClasses https://github.com/nordnet/nordnet-ui-kit/blob/master/docs/testing-component.md#shallow

Comment (JCB-K) [reply test/components/shareville-logo/logo.test.js]: @iamstarkov can't, these are hard-coded classes from the generated svg

File: src/index.js (hunk 1 above)
Comment (JCB-K) [reply src/index.js]: @iamstarkov fixed

**Pull request merged**